ci/socket-io.js

```diff
@@ -36,6 +36,13 @@ var callback = client => {
             ack(Buffer.from([1, 2, 3]));
         }
     });
+
+    client.on('transport_close', data => {
+        console.log(['transport_close', 'Transport close received'])
+        // Close underlying websocket connection
+        client.client.conn.close();
+    })
```

Comment:
nit: switch to verb then subject as we're trying to tell the server to do something, and add a comment saying what it does/why it's here. This is NOT the feature, merely used in testing.
```suggestion
    // This event allows the test framework to arbitrarily close the underlying connection
    client.on('close_transport', data => {
        console.log(['transport_close', 'Transport close received'])
        // Close underlying websocket connection
        client.client.conn.close();
    })
```